web/js/scripts.js

```diff
@@ -102,9 +106,17 @@ var grouphub = (function ($) {
         });
     };
 
+    var updateGroupsInGroup = function () {
+        var $groups = $('.edit_group #group_in_group_tab ul.member_collection');
```

Comment:
Using `data-*` attributes in the DOM on identifiable singular components would be a decent compromise, although I am not familiar enough with the codebase to determine if it is doable in this PR.